src/main/java/org/prebid/server/proto/openrtb/ext/request/ExtPriceGranularity.java

```diff
@@ -12,6 +13,10 @@
 @Value
 public class ExtPriceGranularity {
 
+    public static ExtPriceGranularity from(PriceGranularity priceGranularity) {
```

Comment:
I don't think this method should be placed before fields definition. Usually we place static factory methods after fields definitions in POJOs